adamichelle/wayfarer-api
# #167038690 API consumer should be able to sign in a user

#### What does this PR do?
It creates the endpoint to sign in a user `POST api/v1/auth/signin`

#### Description of Task to be completed?
Have the endpoint to sign in a user working
`POST /api/v1/auth/signin`

#### How should this be manually tested?
After cloning the repo, cd into it and run `npm install`
Create a .env file containing a `DB_URL`, `PRIVATE_KEY` variable which you should set to your database's url and a string or random characters
N.B: You can generate a pair of public and private keys here.
Run `npm run drop` to drop the users table in your db if it exists
Run `npm run create` to create the users table in your db.
Using POSTMAN test the endpoint above with this header:
_key:_ Content-Type _value:_ application/json
